src/BayesFilters/include/BayesFilters/LinearModel.h

```diff
@@ -38,7 +38,7 @@ class bfl::LinearModel : public MeasurementModel
 
     std::pair<bool, bfl::Data> innovation(const bfl::Data& predicted_measurements, const bfl::Data& measurements) const override;
 
-    std::pair<bool, Eigen::MatrixXf> getNoiseSample(const int num) const override;
+    std::pair<bool, Eigen::MatrixXf> getNoiseSample(const int num) const;
```

Comment:
If it does not override, can we move it to `protected`?

Reply:
Done!